gravityflow/gravityflow
# Fix JS Undefined errors with Multiple Status Shortcodes on same page

## Description
Re: [HS#14868](https://secure.helpscout.net/conversation/1291017933/14868/#thread-3724529305)

## Testing instructions
1. Create a page with multiple status shortcodes.
2. Check to note JS errors with the master.
3. Update to this branch, check if the errors are gone and everything works fine.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
